### DISCUSSION TITLE
Residence import

### DISCUSSION THREAD

**Discussion by DJIronic** [GriefPrevention/GriefPrevention#1226] [Ideas]
Hi There!

We would love to migrate from Residence, as the plugin is poorly maintained, developer does not support it.

We will definitely pay for having this plugin as we already paid for Residence. We understand that the dev should receive at least something for supporting his plugin :)

Thanks!

**Comment by RoboMWM** [comment]
Made something for this a while ago. https://github.com/Grief-Prevention/ResidenceToGP releases: https://github.com/Grief-Prevention/ResidenceToGP/releases/tag/1.0

hopefully it still works.
